### DISCUSSION TITLE
Catalog is down

### DISCUSSION THREAD

**Discussion by televisi** [GaetanF/cyborgbackup#82] [Q&A]
Hi,

I'm trying to run the docker instance on my Mac (the end goal is to run the docker instance on QNAP server).

I've followed the instructions on the documentation, as well as:

-  updated the last step to call `api` (as per #55 )
- added "MONGODB_HOST=mongodb" (as per #76)

```
$ wget https://raw.githubusercontent.com/cyborgbackup/cyborgbackup/master/docker-compose.full.yml -O docker-compose.yml
$ cat > .env <<EOF
POSTGRES_PASSWORD=cyborgbackup
POSTGRES_USER=cyborgbackup
POSTGRES_NAME=cyborgbackup
POSTGRES_HOST=postgres
RABBITMQ_DEFAULT_USER=cyborgbackup
RABBITMQ_DEFAULT_PASS=cyborgbackup
RABBITMQ_DEFAULT_VHOST=cyborgbackup
SECRET_KEY=$(openssl rand -base64 47|sed 's/=//g')
MONGODB_HOST=mongodb
EOF
$ docker compose up
$ docker compose exec api /bin/bash
root@72258b8c15db: python3 /cyborgbackup/manage.py loaddata settings
root@72258b8c15db: echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin@cyborg.local', 'admin')"
root@72258b8c15db: python3 /cyborgbackup/manage.py shell
Python 3.7.10 (default, Apr  2 2021, 22:15:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from django.conf import settings
>>> settings.MONGODB_URL
'mongodb://127.0.0.1/'
>>> <CTRL+D>
now exiting InteractiveConsole...
root@72258b8c15db: exit
```

But after I restarted the docker instance/stack, it still showing red catalog system status and blank archives catalog page:
<img width="244" alt="image" src="https://github.com/cyborgbackup/cyborgbackup/assets/4546680/c30e1392-568a-4e33-8d54-2412648e64f2">
<img width="715" alt="image" src="https://github.com/cyborgbackup/cyborgbackup/assets/4546680/662d1962-e755-451b-9cec-55fe3e1aed4f">
 
Any help is greatly appreciated!

Thanks

**Comment by GaetanF** [comment]
Hello,

I have just try the same thing as you and I have the correct value on MONGODB_URL.

Could you check on the API container if the MONGODB_HOST are correctly present on environment variables ?

Also try if Python can correctly handle this ?
```
python3 /cyborgbackup/manage.py shell
>>> import os
>>> os.environ['MONGODB_HOST']
'mongodb'
```

Regards
